tests/management/group/test_view.py

```diff
@@ -1776,8 +1868,6 @@ def test_system_flag_update_on_add(self, send_kafka_message):
             self.assertEqual(self.defGroup.tenant, self.public_tenant)
             self.assertEqual(response.status_code, status.HTTP_200_OK)
 
-            # New platform default role for tenant created
-            custom_default_group = Group.objects.get(platform_default=True, tenant=self.tenant)
             self.assertEqual(custom_default_group.name, "Custom default access")
             self.assertFalse(custom_default_group.system)
             self.assertEqual(custom_default_group.tenant, self.tenant)
```

Comment:
Maybe add an assert to check that the uuid is the same as the default group uuid in the TenantMapping?

Reply:
Added 👍